### ISSUE TITLE
Mapping of ctcss

### ISSUE THREAD

**Issue by cidrblock** [itsmaxymoo/bc125py#3]
I see the bct125a is returning integers for the ctcss values.  Are you aware of a mapping anywhere?

eg, I set the ctcss to 225.7 in the radio and the resulting json/csv has 108.

Great stuff btw, nice clean code too.

```
Reading from scanner...
> /home/bthornto/github/bc125/venv/lib64/python3.11/site-packages/bc125py/sdo.py(720)from_command_response()
-> self.index = int(command_response[0])
(Pdb) command_response
('1', 'Steve net', '04627000', 'AUTO', '108', '2', '0', '1')
(Pdb) 
```

**Comment by itsmaxymoo** [comment]
Yes, there's a mapping performed internally by the radio. It is detailed on the last page of the [protocol specification](https://github.com/itsmaxymoo/bc125py/blob/main/docs/BC125AT_Protocol.pdf) document. I agree it isn't the most user friendly. I'm open to adding support for string parsing or something of the sort, as long as it is implemented cleanly. My only concern would be outputting "human friendly" data *from* the scanner (like `ctcss67.0`, and a user getting confused thinking they can set any value for the ctcss code, even though the scanner only accepts predefined values.

**Comment by cidrblock** [comment]
Thanks, I didn't look there and didn't come across that doc while searching.  I wonder why they did that.....

Let me see if I can familiarize myself with the code enough to get something added. 

I can at least do a PR for the readme so you don't get bothered again by the same question

**Comment by itsmaxymoo** [comment]
I see your PR. I may have some time today, if not definitely in this next week to review. Thank you for your time!

**Comment by itsmaxymoo** [comment]
This feature has been implemented in upcoming version 0.10.0

**Comment by cidrblock** [comment]
Great, I'll look for it.
